recipe/meta.yaml

```
@@ -11,8 +11,10 @@ source:
 build:
   script_env:
     - LIGHTFM_NO_CFLAGS
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 2
+  script: 
+    - export LIGHTFM_NO_CLFAGS=1
```

This doesn't work on Windows: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=270602&view=logs&j=5be07ae1-d8ba-5406-47b6-8e3a3a12f825&t=0bf03e01-0bec-5b85-5316-b1633322e895&l=474

I think you need:

```suggestion
    - export LIGHTFM_NO_CLFAGS=1  # [not win]
    - set LIGHTFM_NO_CLFAGS=1     # [unix]
```

---

I checked the linux build logs and the march=native argument is now gone as expected